runners/core-java/src/main/java/org/apache/beam/runners/core/metrics/MetricsContainerImpl.java

```diff
@@ -85,6 +86,9 @@ public CounterCell getCounter(MetricName metricName) {
     return counters.get(metricName);
   }
 
+  @Nullable
+  public CounterCell tryGetCounter(MetricName metricName) { return counters.tryGet(metricName); }
```

Comment:
Probably worth adding the equivalent for the other get methods (eg., tryGetDistribution).

Reply:
Added tryGetDistribution and tryGetGauge.